x-pack/platform/plugins/shared/ml/server/routes/saved_objects.ts

```diff
@@ -88,6 +88,34 @@ export function savedObjectsRoutes(
             query: syncJobObjects,
           },
         },
+        options: {
+          oasOperationObject: () => ({
```

Comment:
This looks good! Nice work @jgowdyelastic ! Minor suggestion: factorizing these examples into another file might keep the flow of your route definition a bit slimmer. Also, using TypeScript, you can  check that `value` matches what your code returns. Up to you!